src/main/java/Duke.java

```diff
@@ -1,10 +1,214 @@
+import models.Deadline;
+import models.Event;
+import models.Task;
+import models.Todo;
+import io.DukePrint;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class Duke {
+
+    private static final ArrayList<Task> tasks = new ArrayList<>();
+    private static int taskCount = 0;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        DukePrint.printLogo();
+
+        Scanner sc = new Scanner(System.in);
+        String phrase;
+
+        try {
+            loadFile();
+            System.out.println("Save data loaded!");
+        } catch (Exception e) {
+            System.out.println("No save files found.");
+        }
+
+        DukePrint.printDivider();
+        System.out.println("What's up! I'm Duke");
+        System.out.println("What can I do for you?");
+        DukePrint.printEndDivider();
+
+        do {
+            phrase = sc.nextLine();
+            DukePrint.printDivider();
+            inputCommand(phrase, tasks);
+            DukePrint.printEndDivider();
+        } while (!phrase.equals("bye"));
+    }
+
+    public static void inputCommand(String phrase, ArrayList<Task> tasks) {
+        String[] subStrings = phrase.split(" ");
+        int dividerPosition = phrase.indexOf("/");
+
+        switch (subStrings[0]) {
+        case ("bye"):
+            // Exits program
+            System.out.println("Alright cheers mate!");
+            // Saves task list
+            try {
+                saveFile();
+            } catch (Exception e) {
+                System.out.println("Failed to save file!");
+            }
+            break;
+        case ("list"):
+            // List all tasks
+            System.out.println("Here are the tasks in your list:");
+            int i = 0;
+            for (Task task : tasks) {
+                System.out.println(++i + ". " + task.toString());
+            }
+            break;
+        case ("done"):
+            // Set a task as done
+            try {
+                int taskIndex = phrase.charAt(phrase.length() - 1) - '0';
+
+                Task task = tasks.get(taskIndex - 1);
+                task.markAsDone();
+                System.out.println("Nice! I've marked this task as done:");
+                System.out.println("[" + task.getStatusIcon() + "] " + task.getDescription());
+            } catch (NumberFormatException e) {
+                System.out.println("\uD83D\uDE2D Please input a valid number!");
```

Comment:
Perhaps you could save your unicode as a static final constant, to make it easier to see what you're printing